lib/mockRequest.js

```diff
@@ -191,6 +191,47 @@ function createRequest(options) {
         return Accepts.type(types);
     };
 
+    /**
+     * Check if the given `encoding`s are accepted.
+     *
+     * @param {String} ...encoding
+     * @return {String|Array}
+     * @public
+     */
+
+    mockRequest.acceptsEncodings = function(){
+        var accept = accepts(this);
+        return accept.encodings.apply(accept, arguments);
```

Comment:
Explicitly pass named arguments to `accept.encodings`:

```
mockRequest.acceptsEncodings = function(encodings){
        var accept = accepts(mockRequest);
        return accept.encodings(encodings);
};
```

Please do the same for `acceptsCharsets ` and `acceptsLanguages `

Reply:
Thanks for you feedback.

I fixed this too.
Though I'm concerned that rest spread arguments aren't supported now by the mock, though they would work with express.

Reply:
done